plugins/versionpress/composer.json

```diff
@@ -22,7 +22,7 @@
     "symfony/process": "^3.3",
     "michelf/php-markdown": "^1.7",
     "symfony/filesystem": "^3.3",
-    "phpmyadmin/sql-parser": "3.4",
+    "phpmyadmin/sql-parser": "^4",
```

Comment:
Should we better use common entry `^4.3` instead of only `^4` (as all other dependencies in this file)?

Reply:
sql-parser follows semver so all future 4.x versions will work with VersionPress. I don't see any value in restricting the minor version, especially since VP uses a lock file so all dependencies are fixed and any dependency updates must be reviewed / are not automatic. However, if that's what you want to do, I'm happy to make that change to get this PR merged. :)

Reply:
`^4.3` doesn't really restrict the minor version (as you pointed out), it's just a small stylistic request to keep things consistent.

The bigger issue is the `npm install` warning, there are some hints here: https://github.com/bovigo/vfsStream/pull/174.

Reply:
I added a commit to fix that warning.